evaldo/builtins.go

```diff
@@ -8118,7 +8118,7 @@ var builtins = map[string]*env.Builtin{
 	// Tests:
 	// equal { { } .is-empty } 1
 	// equal { dict { } |is-empty } 1
-	// equal { spreadsheet { } { } |is-empty } 1
+	// equal { spreadsheet { 'a 'b } { } |is-empty } 1
```

Comment:
Changed because I used the same column function for all spreadsheet creation operations.